src/Simplex/Messaging/Agent.hs

```diff
@@ -272,12 +272,12 @@ sendConfirmation ::
   SendQueue ->
   SMP.SenderKey ->
   m ()
-sendConfirmation c SendQueue {server, sndId} senderKey = do
+sendConfirmation c sndQ@SendQueue {server, sndId} senderKey = do
```

Comment:
rcvQ and sndQ are used I think - doesn't he warn about shadowing?

Reply:
can be just `rq` or `sq`

Reply:
I don't get warnings.. Where are they used?

Reply:
w/e, changed anyway